controllers/nodehealthcheck_controller.go

```diff
@@ -700,10 +699,11 @@ func (r *NodeHealthCheckReconciler) isControlPlaneRemediationAllowed(ctx context
 	}
 	// if there is a control plane remediation CR for this node already, we can continue with the remediation process
 	for _, cr := range controlPlaneRemediationCRs {
-		if cr.GetName() == node.GetName() {
+		nodeName := utils.GetNodeNameFromCR(cr)
```

Comment:
fixes `[FAIL] Node Health Check CR Reconciliation control plane nodes when two control plane nodes are unhealthy [It] should remediate one after another` (false event)